Puppetfile

```diff
@@ -79,3 +79,8 @@ mod 'datadog/datadog_agent', '1.6.0'
 
 # Used for grabbing certificates for jenkins.io
 mod 'danzilio/letsencrypt', '1.0.0'
+
+# For managing ldap
+mod 'camptocamp/openldap', '1.14.0'
+mod  'herculesteam/augeasproviders_shellvar', '2.x'
```

Comment:
Yes these are used by the openldap module. Note in the commit message we also need to install an augeas package to make the provider functional.